src/main/java/com/alkemy/ong/security/SecurityConfig.java

```diff
@@ -59,9 +59,9 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 .antMatchers("/auth/**", "/h2/**", "/users/**", "/testimonials").permitAll()
                 .and()
-                .antMatcher("/organization/**")
                 .addFilterBefore(jwtTokenFilter(), UsernamePasswordAuthenticationFilter.class)
                 .authorizeRequests()
+                .antMatchers("/organization/**", "/auth/me").permitAll()
```

Comment:
Con el permitAll basicamente le estas diciendo que no le importe el `jwtTokenFilter` de antes, porque dejas pasar a todos 😅 
Te tiene que quedar:
```suggestion
                .antMatchers("/organization/**", "/auth/me").authenticated();
```
y la línea de abajo la podes sacar 👍🏼